test_runner/fixtures/neon_fixtures.py

```diff
@@ -4791,7 +4791,8 @@ def pull_timeline(
         pull_timeline from srcs to self.
         """
         src_https = [f"http://localhost:{sk.port.http}" for sk in srcs]
-        res = self.http_client().pull_timeline(
+        auth_token = self.env.auth_keys.generate_tenant_token(tenant_id)
```

Comment:
please revert the change to this file, it is not needed and thus confusing to readers. We already pass auth tokens when creating the safekeeper http client.